.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,13 +1,2 @@
-arguments=
```

Comment:
can we delete this file ?

Reply:
Yes we can, sorry for the delay, it must be something of my IDE sorry about that.